{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -30,12 +30,17 @@
     path("sitemap.xml", sitemap, name="sitemap"),
 {%- else %}
     path("admin/", admin.site.urls),
+    path("", TemplateView.as_view(template_name="homepage.html")),
 {%- endif %}
     path(
         "robots.txt", TemplateView.as_view(template_name="robots.txt", content_type="text/plain")
     ),
 ]
 
+# Allow testing of all styles locally
+if settings.DEBUG:
+    urlpatterns += [path("demo-styles/", TemplateView.as_view(template_name="demo_styles.html"))]
```

Comment:
I'm not sure if this and `demo_styles.html` should be just `demo` or `demo_content` as this might not just be styles, especially as it grows

Reply:
Tweak this as needed when it comes to actually adding the styles!

I'm sure we'll figure out what templates and demo pages we'll need once we get going with it.